cli/user/README.md

```diff
@@ -94,7 +95,52 @@ All available commands:
 	stream read scoped-stream-name [group-similar] [timeout-in-seconds]: Reads all Events from a Stream and then tails the Stream.
 ```
 
+You can try out commands such as:
+```
+> scope create example
+Scope 'example' created successfully.
+```
+
 For more info on how the Pravega User CLI works, please visit [this page](https://github.com/pravega/pravega/wiki/Pravega-User-CLI).
 
+## Pravega CLI on Kubernetes
+The pravega-cli can be accessed through the kubernetes pod such as the segmentstore pod.
+
+
+Once in the pod, you can run the pravega-cli under admin mode
```

Comment:
If you want to describe how to use the Pravega User CLI from inside the Kubernetes cluster, please say so like I mentioned in the previous README: `From the available options, we now show how to use the Pravega User CLI from inside the Kubernetes cluster, and concretely, by using the built-in tool in the Pravega server instances` (or something like that to give context to the reader).